### PR TITLE
DMP-1625-ARM-CreateMediaManifestFile

### PR DESCRIPTION
**Pull request by karen-hedges**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/DMP-1625

### Change description ###

Fixed issue with transcription and annotations manifest file

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/darts/arm/service/ArchiveRecordServiceImplTest.java
Comment (johnrace0) [inline src/test/java/uk/gov/hmcts/darts/arm/service/ArchiveRecordServiceImplTest.java:1]: test names probably need to change now that the methods are implemented - also not sure if we have a convention for this - but should we have logging messages from tests?

File: src/test/resources/Tests/arm/component/ArchiveAnnotationMetadata/expectedResponse.a360
Comment (johnrace0) [inline src/test/resources/Tests/arm/component/ArchiveAnnotationMetadata/expectedResponse.a360:1]: probably a good reason for it - nut noticed that all these response files are the same

**Review by johnrace0 — CHANGES_REQUESTED** [review]
a few low level observations/comments

File: src/test/java/uk/gov/hmcts/darts/arm/service/ArchiveRecordServiceImplTest.java
Comment (karen-hedges) [reply src/test/java/uk/gov/hmcts/darts/arm/service/ArchiveRecordServiceImplTest.java:1]: I've kept the logging in as the transcription and annotation types need to be fully implemented but the design of these is not yet finalised.

File: src/test/resources/Tests/arm/component/ArchiveAnnotationMetadata/expectedResponse.a360
Comment (karen-hedges) [reply src/test/resources/Tests/arm/component/ArchiveAnnotationMetadata/expectedResponse.a360:1]: The transcription and annotation types have not been fully implemented as the design of these is not yet finalised.

**Review by johnrace0 — APPROVED** [review]

**Pull request merged**